src/dotnet/Users.Service/Module/UsersServiceModule.cs

```diff
@@ -110,7 +111,7 @@ public override void InjectServices(IServiceCollection services)
 
         // Auth services
         var fusionAuth = fusion.AddAuthentication();
-        services.TryAddScoped<ServerAuthHelper>(); // Replacing the default one w/ own
+        services.TryAddScoped<Stl.Fusion.Server.Authentication.ServerAuthHelper, ServerAuthHelper>(); // Replacing the default one w/ own
```

Comment:
I would rather rename local `ServerAuthHelper` to `AppServerAuthHelper`, or something like this :)

Reply:
I.e. you can use `App` prefix to distinguish between own & base types.

Reply:
done